.github/actions/commit-generated-files/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Commit generated files'
+description: 'Automatically commit generated files'
+runs:
+  using: composite
+  steps:
+    - run: |
+        if [ ! -z "$(git status --porcelain)" ]; then  
+          git fetch --depth=1
+          git checkout ${{ github.head_ref }}
+          git add 'packages/atomic/**/*.d.ts' 'packages/atomic-react/src/components/stencil-generated/*'
```

Comment:
What would happen if `git status --porcelain` returned files, but they did not match these paths? Would this lead to empty commits being pushed? errors?

Reply:
Pretty much "nothing".

If you looked at the output in github action you would see something like `No staged files found, nothing to commit working tree clean`

But no errors, and no empty commit.